ubuntu-headless-qt/include/03_prepare_conf.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function set_config use to copy config file to ubuntu os.
```

Comment:
This description looks like a function description. Can you update it and move the old description to the correct location of the mentioned functions?